client/src/lib/graphql/statistics.graphql

```diff
@@ -1,3 +1,18 @@
 query GetStatistics($year: Int!) {
   statistics(year: $year)
+}
+query GetAllCachedStatistics {
+  cachedStatistics {
+    number_of_project
+    link_to_all_projects
+    year
+  }
+}
+
+query GetCachedStatistics($year: Int!) { 
+  cachedStatistics(where: { year: { equals: $year } }) {
```

Comment:
Can we put this in the `GetStatistics` query so that we only need to make one backend request instead of two?